base/util.go

```diff
@@ -139,6 +139,10 @@ func ToInt64(value interface{}) (int64, bool) {
 	return 0, false
 }
 
+func IsPseudoDoc(ID string) bool {
+	return strings.HasPrefix(ID, "_user/") || strings.HasPrefix(ID, "_role/")
```

Comment:
I was looking at func makeChangeEntry changes.go L#180, but there are a few places where ChangeEntry instances are created inline. e.g. sendChangesForDocIds changes_api.go L#408

`row := &db.ChangeEntry{ID: doc.DocID}`

Although the context might make it unlikely to define a pseudo doc.